src/CacheCleaner/AbstractRequestBuilder.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Spatie\ResponseCache\CacheCleaner;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+
+abstract class AbstractRequestBuilder
+{
+    protected string $method = 'GET';
```

Comment:
Could we make all  http methods lowercase in code. Make sure that the code works case insensitive.

Reply:
Request::create() use uppercase Http Method, but AbstractRequestBuilder->setMethod() is case insensitive.